timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/BatchPaxosLearner.java

```diff
@@ -59,7 +61,7 @@
     @Path("learned-values")
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
-    SetMultimap<Client, PaxosValue> getLearnedValues(Set<WithSeq<Client>> clientAndSeqs);
+    SetMultimap<Client, WithSeq<Optional<PaxosValue>>> getLearnedValues(Set<WithSeq<Client>> clientAndSeqs);
```

Comment:
I don't think this is consistent with the javadoc we wrote though:

> Values where nothing has been learnt are excluded. If for a given Client nothing has been learned, the Client is also excluded.

Probably need to either change that to reflect this (in which case I assume that the client/WithSeq values returned form `clientAndSeqs`), or change the implementation so it filters correctly

Reply:
that's fair, I think I'm in favour of filtering it out, it's just cruft otherwise

Reply:
good shout

Reply:
Yeah, also in favour of filtering